registration/registration-services/src/main/resources/spring.properties

```diff
@@ -6,8 +6,9 @@ mosip.optional-languages=
 mosip.min-languages.count=1
 mosip.max-languages.count=1
 
-#Adding default version-mappings here to work as fallback in upgrade scenario, if version-mappings is not configured.
-mosip.registration.verion.upgrade.default-version-mappings={ "1.1.4":{ "dbVersion":"1.1.4", "releaseOrder":1}, "1.1.5":{ "dbVersion":"1.1.5", "releaseOrder":2}, "1.1.5.5":{ "dbVersion":"1.1.5.5", "releaseOrder":3}, "1.2.0":{ "dbVersion":"1.2.0", "releaseOrder":4}, "1.2.0.1-B1":{ "dbVersion":"1.2.0.1", "releaseOrder":5}, "1.2.1-SNAPSHOT":{ "dbVersion":"1.2.1", "releaseOrder":6} }
+#Property to support version upgrade. Version upgrade includes software upgrade and DB upgrade. During the DB upgrade, the DB scripts execution will be done based on the order of versions mentioned here.
+#The name of the backup folder which is created during software upgrade will be used as the previously installed version.
+mosip.registration.verion.upgrade.version-mappings={ "1.1.4":{ "dbVersion":"1.1.4", "releaseOrder":1}, "1.1.5":{ "dbVersion":"1.1.5", "releaseOrder":2}, "1.1.5.5-P1":{ "dbVersion":"1.1.5.5", "releaseOrder":3}, "1.2.0.1-SNAPSHOT":{ "dbVersion":"1.2.0.1", "releaseOrder":4} }
```

Comment:
@HimajaDhanyamraju2  version map is missing the current released version and beta versions

Reply:
Added beta version to the versions map. Please check now.